pkg/sql/tablemetadatacache/update_table_metadata_cache_job.go

```diff
@@ -38,13 +38,19 @@ var _ jobs.Resumer = (*tableMetadataUpdateJobResumer)(nil)
 
 // Resume is part of the jobs.Resumer interface.
 func (j *tableMetadataUpdateJobResumer) Resume(ctx context.Context, execCtxI interface{}) error {
-	log.Infof(ctx, "starting table metadata update job")
 	j.job.MarkIdle(true)
 
 	execCtx := execCtxI.(sql.JobExecContext)
 	metrics := execCtx.ExecCfg().JobRegistry.MetricsStruct().
 		JobSpecificMetrics[jobspb.TypeUpdateTableMetadataCache].(TableMetadataUpdateJobMetrics)
-
+	var onJobStartKnob func()
+	var onJobCompleteKnob func()
```

Comment:
If you want you can combine this into one line
var onJobStartKnob, onJobCompleteKnob, onJobReady func()

Reply:
Done